lnrpc/routerrpc/router.proto

```diff
@@ -389,8 +389,8 @@ message PairData {
     /// Time stamp of last result.
     int64 timestamp = 1 [json_name = "timestamp"];
 
-    /// Minimum penalization amount (only applies to failed attempts).
-    int64 min_penalize_amt_sat = 2 [json_name = "min_penalize_amt_sat"];
+    /// Last amount that was forwarded.
+    int64 amt_sat = 2 [json_name = "min_penalize_amt_sat"];
```

Comment:
should we make this msat? Since msat is used internally, it could lead to some confusion in the rounding case.

Reply:
added msat and kept sat